front/src/components/AdminPanel/Panel/Competitors/Competitors.js

```diff
@@ -1,8 +1,204 @@
+                                className="form-control"
+                                value={surname}
+                                onChange={(event) => setSurname(event.target.value)}
+                            />{" "}
+                            <br />
+                            <input
+                                type="number"
+                                placeholder="Numer licencji"
+                                className="form-control"
+                                value={licenceNumber}
+                                onChange={(event) => setlicenceNumber(event.target.value)}
+                            />{" "}
+                            <br />
+                            <input
+                                type="number"
+                                min = "1920"
+                                max = "2020"
+                                placeholder="Rocznik"
+                                className="form-control"
+                                value={year}
+                                onChange={(event) => setYear(event.target.value)}
+                            />{" "}
+                            <br />
+                            <input
+                                type="text"
+                                placeholder="Klub"
+                                className="form-control"
+                                value={club}
+                                onChange={(event) => setClub(event.target.value)}
+                            />{" "}
+                            <br />
+                            <Button
+                                type="submit"
+                                placeholder="Dodaj zawodnika"
+                                className="btn btn-primary btn-lg"
+                            />
+                        </form>
+                    </div>
+                </div>
+                    <div className="competitors-right-div">
+                        <div className="competitors-tile-div">
+                            <p className="competitors-header"> Lista zawodników spełniających kryteria </p>
+                            <table className="table table-hover" style={{marginLeft: "2%"}, {width:"90%"}}>
+                                    <tbody>
+                                    {
+                                        Object.keys(competitors).map(function(element) {
+                                            return <tr className={chosenCompetitor == competitors[element]['id'] && "activeRow"}> <td> <input type="radio" name="competitorName" value={competitors[element]['id']} onChange={(event) => setChosenCompetitor(event.target.value)}/> 
+                                            {competitors[element]['name']} {competitors[element]['surname']}, {competitors[element]['gender'] == "kobieta" ? 'K' : 'M'}, {competitors[element]['year']}, {competitors[element]['licenceNumber']}  </td></tr>
```

Comment:
Is "kobieta" standardized somewhere? Set automatically? In this review alone I see "kobieta" and "Kobieta" interleaved (same story with "mężczyzna" and "Mężczyzna").
You should consider transforming string to lowercase before comparison to avoid bugs in such cases.